iroha_p2p/src/lib.rs

```diff
@@ -27,6 +27,15 @@ pub enum Error {
     /// Failed to create keys
     #[error("Failed to create session key")]
     Keys,
+    /// Failed to parse address
+    #[error("Failed to parse socket address")]
+    Addr,
+}
+
+impl From<AddrParseError> for Error {
+    fn from(_: AddrParseError) -> Self {
+        Self::Addr
+    }
```

Comment:
Is there a reason to discard addr parse error?

Reply:
Changed the type of `Error::Addr`, removed this impl.